src/components/Equation.js

```diff
@@ -65,17 +55,28 @@ export class Equation extends React.Component {
     let widgets;
 
     const attribs = []
-    for (const key in state) {
+    for (const key of Object.keys(state)) {
```

Comment:
I dont' think we need the `Object.keys`

Reply:
using `of` to iterate through objects is preferred over `in` since `in` will also iterate over the prototype’s own instances. Usually you’ll see an `isInstanceOf` check when using `in` to traverse, so this is a little safer and cleaner.